firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -73,53 +75,86 @@ class DictionaryEntryDetailSerializer(serializers.HyperlinkedModelSerializer):
     split_words = serializers.SerializerMethodField()
     split_words_base = serializers.SerializerMethodField()
 
-    @staticmethod
-    def get_split_chars(entry):
-        alphabet = Alphabet.objects.filter(site=entry.site).first()
-        ignored_characters = alphabet.ignorable_characters.values_list(
-            "title", flat=True
-        )
+    logger = logging.getLogger(__name__)
+
+    def retrieve_model_or_context(self, model_name):
```

Comment:
Nitpick: this method name is not really accurate anymore. Maybe rename to get_model_from_context ?

Reply:
Good catch, thanks. I have updated the function name.